core/sql/sqlcomp/CmpSeabaseDDLrepos.cpp

```diff
@@ -516,6 +585,62 @@ short CmpSeabaseDDL::upgradeReposComplete(ExeCliInterface * cliInterface,
 
         case 1:
           {
+            // If there were views on the old Repository tables, they are
+            // still there, by virtue of the fact that we did "SKIP VIEW CHECK"
+            // on the ALTER TABLE RENAME. Now we will capture their view
+            // definitions (which will contain the old table name, not the
+            // renamed table name) and replay that against the new Repository
+            // table. If the replay fails, we treat that as an unrecoverable
+            // situation and ignore it. Instead, we'll save the view definition
+            // text in the TEXT table with a different TEXT_TYPE and clue the
+            // user in that it is there. They can then try to create the view
+            // at their leisure. Note that they may have to change the view
+            // definition themselves, e.g. if we dropped a column from the
+            // repository table that their view happened to reference.
+            //
```

Comment:
Metadata tables store static information about objects. They dont usually
contain status or error information about user action.

Looks like TEXT table is being overloaded to contain metadata related information
as well as error log status.

Failed actions are logged in log files. install may be putting them in error logs.
Another example of logging error information is when failed rows during load are logged into error logs that users can query later.

Upgrade is a platform only operation that is done as part of install/upgrade.
Would it be better to put failed operations in an error log that can be queried
or looked at later.

Maybe at some point a repository error log table can be added that contains  information
about failed operations. And this table is viewable by everyone.